File: default_output/flutter/person_like/person_like_migration.dart
class Database
{
    void execute(String s) {}
}

class PersonLikeMigration
{
    void createDatabase(Database db)
    {
        db.execute("CREATE TABLE PersonLike"
              +"person_like_uuid TEXT, "
              +"liked_user_uuid TEXT, "
              +"liking_user_uuid TEXT"
            ")"
        );
    }
}

File: default_output/phoenix/person_like/person_like_migration.ex
defmodule FinalLocallity.Repo.Migrations.CreatePersonLike do
  use Ecto.Migration

  def change do
    create table(:person_like) do
      add :person_like_uuid, :string
      add :liked_user_uuid, :string
      add :liking_user_uuid, :string

      timestamps()
    end

  end
end
